src/components/redux/books/books.js

```diff
@@ -0,0 +1,40 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
[OPTIONAL] You can make them like `const ADD_BOOK = bookstore/books/ADD_BOOK and const REMOVE_BOOK = bookstore/books/REMOVE_BOOK` to follow the ducks pattern.

Reply:
yes sure ,
thank you 😄